app/controllers/admin/reservations/item_pool_searches_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Admin
+  module Reservations
+    class ItemPoolSearchesController < BaseController
+      include Pagy::Backend
+
+      def show
+        item_pool_scope = ItemPool.all
+
+        if params[:category]
+          @category = CategoryNode.where(id: params[:category]).first
+          redirect_to(items_path, error: "Category not found") && return unless @category
+
+          item_pool_scope = @category.item_pools
+        end
+
+        item_pool_scope = item_pool_scope.includes(:categories).order("name ASC")
```

Comment:
```suggestion
        item_pool_scope = item_pool_scope.includes(:categories).order(name: :asc)
```
Similar to the above. I've been on teams that enforce it one way or the other so it jumps out at me 😅

Reply:
I don't think we've been too strict about this one way or the other so far... if there's a linter we could use to enforce this kind of thing, that would be nice!